pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/RuntimeContainerBase.groovy

```diff
@@ -0,0 +1,155 @@
+import org.jenkinsci.plugins.pipeline.StageStatus
+import org.jenkinsci.plugins.pipeline.StageTagsMetadata
+import org.jenkinsci.plugins.pipeline.SyntheticStage
+import org.jenkinsci.plugins.pipeline.modeldefinition.actions.DeclarativeJobAction
+import org.jenkinsci.plugins.pipeline.modeldefinition.actions.DeclarativeJobPropertyTrackerAction
+import org.jenkinsci.plugins.pipeline.modeldefinition.actions.ExecutionModelAction
+import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTBuildParameter
+import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTMethodCall
+import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTOption
+import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTTrigger
+import org.jenkinsci.plugins.pipeline.modeldefinition.causes.RestartDeclarativePipelineCause
+import org.jenkinsci.plugins.pipeline.modeldefinition.model.Environment
+import org.jenkinsci.plugins.pipeline.modeldefinition.model.StepsBlock
+import org.jenkinsci.plugins.pipeline.modeldefinition.options.DeclarativeOption
+import org.jenkinsci.plugins.pipeline.modeldefinition.options.impl.QuietPeriod
+import org.jenkinsci.plugins.pipeline.modeldefinition.steps.CredentialWrapper
+import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.Whitelisted
+import org.jenkinsci.plugins.structs.SymbolLookup
+import org.jenkinsci.plugins.structs.describable.DescribableModel
+import org.jenkinsci.plugins.structs.describable.UninstantiatedDescribable
+import org.jenkinsci.plugins.workflow.actions.LabelAction
+import org.jenkinsci.plugins.workflow.actions.NotExecutedNodeAction
+import org.jenkinsci.plugins.workflow.actions.TagsAction
+import org.jenkinsci.plugins.workflow.actions.ThreadNameAction
+import org.jenkinsci.plugins.workflow.cps.CpsFlowExecution
+import org.jenkinsci.plugins.workflow.cps.CpsScript
+import org.jenkinsci.plugins.workflow.cps.CpsThread
+import org.jenkinsci.plugins.workflow.flow.FlowExecution
+import org.jenkinsci.plugins.workflow.graph.BlockEndNode
+import org.jenkinsci.plugins.workflow.graph.BlockStartNode
+import org.jenkinsci.plugins.workflow.graph.FlowNode
+import org.jenkinsci.plugins.workflow.graphanalysis.*
+import org.jenkinsci.plugins.workflow.job.WorkflowJob
+import org.jenkinsci.plugins.workflow.job.WorkflowRun
+import org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty
+import org.jenkinsci.plugins.workflow.steps.FlowInterruptedException
+import org.jenkinsci.plugins.workflow.steps.Step
+import org.jenkinsci.plugins.workflow.steps.StepDescriptor
+import org.kohsuke.accmod.Restricted
+import org.kohsuke.accmod.restrictions.NoExternalUse
+
+import javax.annotation.CheckForNull
+import javax.annotation.Nonnull
+import javax.annotation.Nullable
+import javax.lang.model.SourceVersion
+import java.util.concurrent.TimeUnit
+
+/**
+ *
+ *
+ * @author Liam Newman
+ */
+@SuppressFBWarnings(value="SE_NO_SERIALVERSIONID")
+class RuntimeContainerBase {
+
+    protected static CpsScript that
```

Comment:
Spotbugs wants you to restrict the visibility of this member (not sure if it is looking at Groovy directly or at the Java code generated from the Groovy code).

> RuntimeContainerBase.that should be package protected [org.jenkinsci.plugins.pipeline.modeldefinition.RuntimeContainerBase] In RuntimeContainerBase.groovy MS_PKGPROTECT

It looks like it is only used in this file and could be private.

```suggestion
    private static CpsScript that
```

Reply:
Got it.